docs/validators/setup/cli.md

```diff
@@ -1,60 +1,51 @@
 ---
-description: Build a CLI tool and compile a node binary from source.
-title: Build a Ronin CLI
+description: Install the Ronin CLI and compile a node binary from source.
+title: Build your node using Ronin CLI
 tags:
   - cli
 ---
 
-## Overview
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-This guide demonstrates how to build a Ronin CLI (command-line tool) and compile the node binary on your own instead of using a packed binary from Docker. With the CLI tool, you can install the Ronin node and configure it as a non-validator node (default), validator node, or archive node.
+## Overview
 
-## Before you start
+This guide demonstrates how to compile a Ronin node binary on your own using the Ronin CLI, instead of running a packed binary from Docker. With the CLI, you configure and run a full (non-validator) node, validator node, or archive node.
 
-To build the Ronin CLI, you need to install the following dependencies:
+## Prerequisites
```

Comment:
Let me check Prerequisites here, some guys got issue related to missing lib before running Make step.